### DISCUSSION TITLE
More consistency changes for covers.

### DISCUSSION THREAD

**Discussion by illuzn** [Blackymas/NSPanel_HA_Blueprint#1662] [Ideas]
Given we've just made covers show their correct icons based on device_class.

How about making the control buttons on the cover page do the same? It looks to me like those are hardcoded in the HMI right now?

We could then use a device_class lookup and use garage-opening and garage-closing as the relevant buttons.

**Comment by edwardtfn** [comment]
I liked the idea. Could you please add it as a Enhancement request in the [Issues](https://github.com/Blackymas/NSPanel_HA_Blueprint/issues) area?

**Comment by edwardtfn** [comment]
Moved to issues:
- #1688
